emission/exportdata/export_data.py

```diff
@@ -0,0 +1,50 @@
+# Standard imports 
+import logging
+
+# Our imports
+import emission.storage.pipeline_queries as epq
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.decorations.analysis_timeseries_queries as esda
+
+import gzip
+import json
+import bson.json_util as bju
+
+def set_export_data(user_id):
+    time_query = epq.get_time_range_for_export_data(user_id)        # Does this have .startTs and .endTs atributes? Should we use those?
```

Comment:
Not sure what your comment here means. Looking at your implementation of `get_time_range_for_export_data`, 

```
tq.timeType = "data.end_ts"
```

So it does already use `end_ts`, right?